Docker/DockerfileLaravelProd

```diff
@@ -1,4 +1,7 @@
-FROM php:8-fpm
+FROM php:8.2-apache
```

Comment:
```suggestion
FROM php:8.2-fpm
```

I think we should prefer to keep using `fpm`. https://www.php.net/manual/en/install.fpm.php